src/hathor.c

```
@@ -30,6 +35,10 @@ void hash160(uint8_t *in, size_t inlen, uint8_t *out) {
     } u;
     uint8_t buffer[32] = {0};
 
+    if ((in == NULL) || (out == NULL)) {
+        return;
```

Are failing silently here?